R/package_skeleton.R

```diff
@@ -41,20 +41,23 @@ package_skeleton <- function(path = ".", git_init = TRUE, git_clone = NULL,
     git_init = FALSE
   }
   
-  if (description_template ) {
-    add_description(npath)
-  }
+
+  bcgovr_desc = list("Package" = sub('.*\\/', '', npath))
   
   ## Add in package setup files
-  devtools::setup(npath, rstudio = FALSE) 
+  devtools::setup(npath, rstudio = TRUE, description = bcgovr_desc) 
```

Comment:
Only that not everyone uses RStudio... although the number of those that don't is getting smaller